<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meteo</class>
 <widget class="QDialog" name="meteo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(95, 95, 95);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>841</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Small Fonts</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 0, 4);</string>
   </property>
   <property name="text">
    <string>Vous pouvez visualisez certaines grandeurs physiques mesurées par votre sation météo pour une date donnée</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Entrez la date </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>température</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>humidité</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">color :rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Vitesse du vent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Direction du vent</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="data">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>180</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="placeholderText">
       <string>AAAA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string>MM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string>JJ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(5, 126, 255);
font: 18pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>540</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="date">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>130</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 700 italic 20pt &quot;Consolas&quot;;
padding :  0% auto;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>160</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>date</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>img</zorder>
  <zorder>data</zorder>
  <zorder>verticalGroupBox</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>widget</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
